packages/reports/src/Containers/open-positions.jsx

```diff
@@ -309,10 +310,28 @@ const OpenPositions = ({
     server_time,
     ...props
 }) => {
-    const [active_index, setActiveIndex] = React.useState(is_multiplier ? 1 : 0);
     // Tabs should be visible only when there is at least one active multiplier contract
     const [has_multiplier_contract, setMultiplierContract] = React.useState(false);
+    const [contract_type_value, setContractTypeValue] = React.useState('Options');
     const previous_active_positions = usePrevious(active_positions);
+    const contract_types = [
+        {
+            text: localize('Options'),
+            value: 'Options',
+        },
+        {
+            text: localize('Multipliers'),
+            value: 'Multipliers',
+        },
+        {
+            text: localize('Vanilla'),
+            value: 'Vanilla',
+        },
+        {
+            text: localize('Accumulators'),
+            value: 'Accumulators',
+        },
+    ];
```

Comment:
@henry-deriv please remove this contract_types object. We have to keep existing logics but replace Tabs with Dropdown. Turbos or Vanillas will be added to the dropdown by their respective project teams later.

Reply:
Thanks, @maryia-deriv. Will do!